src/components/menu-top/MenuTop.vue

```diff
@@ -40,6 +40,22 @@
               Four step grid
             </b-checkbox>
           </b-navbar-item>
+          <b-navbar-item>
+            <b-checkbox
+              v-model="yardlines"
+              data-test="menu-top--view-yardlines"
+            >
+              Yardlines
```

Comment:
Just realized, but in the dictionary, it is "yard line" not "yardline" haha https://www.thefreedictionary.com/yard+line

Reply:
oh no will update!